### DISCUSSION TITLE
Cost Index x10 - Simbrief

### DISCUSSION THREAD

**Discussion by StefNx** [flybywiresim/aircraft#2682] [Ideas]
Since on this system the cost index works in undreds and not tens e.g. CI 1 to CI 999 and simbrief instead works from CI 1 to CI 99
it would be nice if, when pulling data from SimBrief it automatically inserted the correct CI multiplying the simbrief data x10 e.g. simbrief= CI 9 - A32NX = CI 90

**Comment by derl30n** [comment]
I don't think that's how it works. The different CIs have different meanings.
